credentials/generate-revocation-set.py

```diff
@@ -91,6 +91,101 @@ def parse_vid_pid_from_distinguished_name(distinguished_name):
     return vid, pid
 
 
+def get_akid(cert: x509.Certificate) -> bytes:
```

Comment:
Since you are catching the exception, the `bytes` output should be optional here.